src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -164,10 +161,8 @@ public abstract class SingleFeaturePredictionTransformerBase<TModel, TScorer> :
         /// </summary>
         public ColumnType FeatureColumnType { get; }
 
```

Comment:
Why not making this internal?

Reply:
I think it's fine actually, disregard that sorry.

---
In reply to: [254558228](https://github.com/dotnet/machinelearning/pull/2453#discussion_r254558228) [](ancestors = 254558228)

Reply:
Do you mean the class? I think this has to remain public.

---
In reply to: [254560344](https://github.com/dotnet/machinelearning/pull/2453#discussion_r254560344) [](ancestors = 254560344,254558228)